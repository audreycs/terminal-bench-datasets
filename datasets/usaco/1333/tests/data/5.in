10
681404 333428
181650 647050
229028 687782
657455 275098
58398 718183
365468 24732
503720 479173
556360 20233
694125 203877
617154 619370
