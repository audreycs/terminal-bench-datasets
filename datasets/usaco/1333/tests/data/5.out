95
141
363
54
148
60
88
232
85
564
